XcodeServerSDK/Utils/HTTPUtils.swift

```diff
@@ -30,15 +30,13 @@ public class HTTP {
             //try to cast into HTTP response
             if let httpResponse = response as? NSHTTPURLResponse {
                 
-                if error != nil {
-                    
+                guard error != nil else {
```

Comment:
When it comes to changes, I think you want to change this line to `guard error == nil else {`, so that we guard against an error being present (in which case we want the `else` block to execute and return early.